app/controllers/admin/site_settings_controller.rb

```diff
@@ -9,40 +9,71 @@ def update
     authorise Setting.all.first
 
     all_updated = update_settings( settings_params )
-    if all_updated
+    if all_updated.nil?
```

Comment:
Having `all_updated == false` and `all_updated == nil` mean two different things feels a bit risky.

Perhaps `#update_settings` could return a result that more clearly defines its possible values, such as a symbol: `:unchanged`, `:success`, `:failure`.